### PR TITLE
[Cloud Security] Add GCP rule templates

### PR DESCRIPTION
**Pull request by orouz**
## What does this PR do?

Adds rule templates

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ]

## Related issues

- Closes https://github.com/elastic/csp-security-policies/issues/305

### CODE REVIEW

File: packages/cloud_security_posture/changelog.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@
     - description: Add mapping to CSPM org fields
       type: enhancement
       link: https://github.com/elastic/integrations/pull/7818
+    - description: Add gcp rule templates
+      type: enhancement
+      link: https://github.com/elastic/integrations/pull/8007
```
Comment (orouz) [inline packages/cloud_security_posture/changelog.yml:21]: @oren-zohar should i also bump the version (like `1.6.0-preview14`) and then use it in the manifest?

**Comment by orouz** [comment]
looks like `elastic-package-stack-kibana-1 ` container has errors and exits which [fails](https://fleet-ci.elastic.co/blue/organizations/jenkins/Ingest-manager%2Fintegrations/detail/PR-8007/4/pipeline#step-1210-log-93) the CI

**Comment by orouz** [comment]
@jlind23 

we need this for `8.11.0` as it includes additional CSP rule templates we now support. these changes are generated by a script we use and do not pose a risk. 
since FF is close, we'd prefer not wait for a new `elastic-package` release and instead force-merge this by admin.

**Review by oren-zohar — APPROVED** [review]

**Comment by jlind23** [comment]
@mrodm @jsoriano looks like the same unrelated problem we have already seen. When it the new elastic package available? Or shall we just force merge anyway?

**Comment by mrodm** [comment]
> @mrodm @jsoriano looks like the same unrelated problem we have already seen. When it the new elastic package available? Or shall we just force merge anyway?

There is already a new elastic-package version v0.88.0 published. In this PR, we're trying to update it for the integrations repository
https://github.com/elastic/integrations/pull/8039

Could it be wait for merging that PR ?

**Comment by mrodm** [comment]
@orouz PR updating to the latest elastic-package has been merged, could you try to re-run the tests or update with main branch ? That should fix this issues related to the failure in kibana container.

**Pull request merged**